### DISCUSSION TITLE
Display UTF-8 characters in console

### DISCUSSION THREAD

**Discussion by linusqzdeng** [kdbinsidebrains/plugin#78] [Q&A]
Hi, thanks for the great work of this plugin! The experience is great except for the display of utf-8 encoded characters in KDB console. I am a Chinese user and would like to know if there are any method to correctly decode Chinese? It can be solved in the command line by changing code page, but I don't know how to implement it in the console.

![image](https://github.com/kdbinsidebrains/plugin/assets/68272577/2b8e60a0-776c-4dd5-8b2f-30929d3e44d3)

**Comment by smklimenko** [comment]
Hi,

Glad you are happy with it. I don't think there is any issues with the plugin here. If I get official Unicode example from KX site that contains exactly Chinese chars (https://code.kx.com/q/kb/unicode/) and run it in the console, I get correct result:

![image](https://github.com/kdbinsidebrains/plugin/assets/3913096/e086b5f6-ac76-41f9-9f19-002dc95c638f)

another sample. If I get Han Character (https://www.fileformat.info/info/unicode/char/6bcf/index.htm) that has UTF-8 hex code: 0xE6 0xAF 0x8F (e6af8f) and convert it to string, I see correct result:

![image](https://github.com/kdbinsidebrains/plugin/assets/3913096/52e5c143-a216-48f4-b2e6-29864c0c9448)

Could you please try same sample in your workspace and if it works - something wrong with your data in the table.
